forte/processors/data_augment/algorithms/embedding_similarity_augmenter.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from typing import Dict, List
+import numpy as np
+from tensorflow import gfile
+from texar.tf.data import Embedding, load_glove
+from forte.processors.data_augment.algorithms.base_augmenter import BaseDataAugmenter
+
+__all__ = [
+    "EmbeddingSimilarityAugmenter",
+]
+
+random.seed(0)
```

Comment:
Please put this seed setup into the class/function scope, and you may allow users to provide seeds.